java/core/src/java/org/apache/orc/impl/ConvertTreeReaderFactory.java

```diff
@@ -1832,7 +1881,6 @@ private static TypeReader createDoubleConvertTreeReader(int columnId,
                                                           TypeDescription fileType,
                                                           TypeDescription readerType,
                                                           Context context) throws IOException {
-
```

Comment:
Please don't touch irrelevant code, @pgaref . We usually don't mix bug fixes and style fixes. For the style fix, you had better do in a separate PR.